README.md

```diff
@@ -1,2 +1,30 @@
-# intel-ffmpeg-patch
-patches set for ffmpeg-cartwheel
+FFmpeg Unmerged Patches
+========================
+Intel upstream patches queue for ffmpeg
+
+ffmpeg mainline codebase
+commit: e71d73b09652f4fc96e512a7d6d4c2ab41860f27
+
+How to use the Unmerged Patches:
+1), How to update patches
+    Please put patch set (*.patch) in your ffmpeg mainline folder.
+
+2), How to submit pull request
+    Please submit pull request for this repo (https://github.com/intel-media-ci/intel-ffmpeg-patch).
+
+3), How to apply patches
+    $ git am *.patch
+    or
+    $ for i in *.patch; do patch -p1 < $i; done
+
+
+Known issues:
+1, ffmpeg-vaapi vp8 decode multi-thread HWAccel decode error
+issue number:
+    https://github.com/intel/media-driver/issues/1028
+    https://github.com/intel/media-driver/issues/317
+ffmpeg patch fixing:
+    https://git.ffmpeg.org/gitweb/ffmpeg.git/commit/eb6bb8f32fdc9be89cce65869dce9dd950e91be2
```

Comment:
The issue has been fixed in FFmpeg, you'd be better to update the code base.

Reply:
Ok, updated